frontend/packages/ceph-storage-plugin/src/components/bucket-class/wizard-pages/namespace-store-pages/multi-namespace-store.tsx

```diff
@@ -23,22 +23,26 @@ export const MultiNamespaceStorePage: React.FC<MultiNamespaceStoreProps> = React
     const openModal = () => NamespaceStoreModal({ namespace });
     const [selectedCount, setSelectedCount] = React.useState(state.readNamespaceStore.length);
     const [enabledItems, setEnabledItems] = React.useState([]);
+    const [selectedItem, setSelectedItem] = React.useState('');
 
     React.useEffect(() => {
       const readItems = state.readNamespaceStore.map(getName);
       setEnabledItems(readItems);
-    }, [state.readNamespaceStore]);
+      setSelectedItem(getName(state.writeNamespaceStore[0]) || '');
+    }, [state.readNamespaceStore, state.writeNamespaceStore, selectedItem]);
```

Comment:
Line 45, is already updating selectedItem, you are overriding same value multiple times

Reply:
this is needed in edit-bucket class where we need to immediately populate the fields on the very first render. I can do it here or above. I've moved it above now